awx/main/models/ha.py

```diff
@@ -147,6 +147,13 @@ def is_isolated(self):
         return self.rampart_groups.filter(controller__isnull=False).exists()
 
     def refresh_capacity(self):
+        if settings.IS_K8S:
```

Comment:
This means that when `IS_K8S`=True`, `/api/v2/ping/` will report 0 capacity for all instances, and the only way to run a playbook is via container group-based execution.